app/views/handlers/feedback.py

```diff
@@ -174,3 +200,41 @@ def is_limit_reached(session_data: SessionData) -> bool:
     @staticmethod
     def is_enabled(schema: QuestionnaireSchema) -> bool:
         return schema.get_submission().get("feedback")
+
+
+class FeedbackMetadata:
+    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
+        self.feedback_count = feedback_count
+        self.form_type = form_type
+        self.language_code = language_code
+        self.region_code = region_code
+        self.tx_id = tx_id
+
+    def __call__(self) -> Mapping:
+        return {
+            "feedback_count": self.feedback_count,
+            "form_type": self.form_type,
+            "language_code": self.language_code,
+            "region_code": self.region_code,
+            "tx_id": self.tx_id,
+            "feedback_submission_date": datetime.utcnow().strftime("%Y-%m-%d"),
```

Comment:
It think this should be a iso format timestamp and not just a date, it's common practice to include date and time. Will also align with other payloads like IR and email.

Reply:
It's supposed to be just a date and matches the requirements as stated on big data card

Reply:
I dont see a reason not to be datetime but fine lets not hold thing up any longer.